README.md

```diff
@@ -119,6 +119,12 @@ machine. Similarly, because our
 MIDI port, it will not work within the Docker container. You'll need to use the
 full Development Environment.
 
+You may find at some point after installation that your Docker image is out of date. To update the image to the latest version, run:
```

Comment:
I would clarify that getting out of date happens when we release new versions of Magenta.